content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -417,6 +417,86 @@ kubectl patch deployment patch-demo --patch-file patch-file.json
 kubectl patch deployment patch-demo --patch '{"spec": {"template": {"spec": {"containers": [{"name": "patch-demo-ctr-2","image": "redis"}]}}}}'
 ```
 
+### Update an object's replica count using `kubectl patch` with `--subresource` {#scale-kubectl-patch}
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+The flag `--subresource=[subresource-name]` is used with kubectl commands like get, patch, edit and replace to fetch and update `status` and `scale` subresources of the resources (applicable for kubectl version v1.24 or more). This flag is used with all the API resources (built-in and CRs) which has `status` or `scale` subresource. Deployment is one of the examples which supports these subresources.
```

Comment:
PLEASE WRAP THE LONG LINES, e.g.

```suggestion
The flag `--subresource=[subresource-name]` is used with kubectl commands like get, patch,
edit and replace to fetch and update `status` and `scale` subresources of the resources
(applicable for kubectl version v1.24 or more). This flag is used with all the API resources
(built-in and CRs) which has `status` or `scale` subresource. Deployment is one of the examples
which supports these subresources.
```

Reply:
I will update this as per your comment soon.

Reply:
addressed